### ISSUE TITLE
does not seem to work with mocha

### ISSUE THREAD

**Issue by matthiasg** [alonbardavid/coffee-script-mapped#5]
Given the mocha test below i see an error in line 5, though it should be line 3. Mocha was started in a test.js with mocha.run() and in the first line "coffee-script-mapped" was included with require.

test.coffee

```
describe 'test', ->
  it 'fails', ->
    throw new Error() 
```

i get:

```
  .

 0 passing (5 ms)
1 failing

1) test fails:
  ReferenceError: t is not defined
    at Context.<anonymous> (c:\Development\dictation-fix-server\test\fail.coffee:5:14)
    at Test.Runnable.run (c:\Development\dictation-fix-server\node_modules\mocha\lib\runnable.js:211:32)
    at Runner.runTest (c:\Development\dictation-fix-server\node_modules\mocha\lib\runner.js:355:10)
    at c:\Development\dictation-fix-server\node_modules\mocha\lib\runner.js:401:12
    at next (c:\Development\dictation-fix-server\node_modules\mocha\lib\runner.js:281:14)
    at c:\Development\dictation-fix-server\node_modules\mocha\lib\runner.js:290:7
    at next (c:\Development\dictation-fix-server\node_modules\mocha\lib\runner.js:234:23)
    at Object._onImmediate (c:\Development\dictation-fix-server\node_modules\mocha\lib\runner.js:258:5)
    at processImmediate [as _immediateCallback] (timers.js:317:15)
```

**Comment by alonbardavid** [comment]
I cannot reproduce it. Can you post the code you used to run Mocha programatically?

I've used the following snippet and it works:

```
 require('coffee-script-mapped') 
 var Mocha = require('Mocha')
 var mocha = new Mocha()
 mocha.addFile("mochaTest.coffee")
 mocha.run()
```

When you use mocha in the commandline it also works:
mocha --compilers coffee:coffee-script-mapped <test>

**Comment by matthiasg** [comment]
We are using https://github.com/curasystems/test-runner... basically we write a **test.js** which consists of require('cura-test-runner')

the test folder contains the tests..

we are also getting inconsistent results in some other projects, but we are still investigating.

**Comment by alonbardavid** [comment]
I've setup a new project.
I've npm installed cura-test-runner.
I've created a folder called tests with a test.coffee file containing only the code in the original comment.
I've created a file in the root folder named runme.js containing only "require ('cura-test-runner')"
I now run node runme.js

it outputs the correct error location.
Is there something else I'm missing in order to recreate the issue?
